src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  filter: Filter,
+  onFilter: (filter: Filter) => void,
+};
+
+export const Footer: React.FC<Props> = ({ filter, onFilter }) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        3 items left
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
```

Comment:
nb: you can reduce your code, if you move some general keys to array and render this links with `.map`